engines/dfc_provider/app/controllers/dfc_provider/application_controller.rb

```diff
@@ -16,6 +16,12 @@ class ApplicationController < ActionController::Base
 
     private
 
+    def require_permission(scope)
+      return true if current_user.is_a? Spree::User
+
+      current_user.permissions(scope).where(enterprise: current_enterprise).exists?
+    end
```

Comment:
❓This method just returns a boolean value. Shouldn't it fire `unauthorized` if false?